scripts/start.sh

```diff
@@ -67,6 +67,21 @@ if [ -n "${RCON_PORT}" ]; then
     sed -i "s/RCONPort=[0-9]*/RCONPort=$RCON_PORT/" /palworld/Pal/Saved/Config/LinuxServer/PalWorldSettings.ini
 fi
 
+if [[ -n "${BACKUP_ENABLED}" ]]; then
```

Comment:
this needs to be if BACKUP_ENABLED = true, now it turns backups on even if the status is false